samples/shield_mcp3561r/src/main.c

```diff
@@ -0,0 +1,96 @@
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/adc.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/printk.h>
+
+#include <inttypes.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/adc.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/util.h>
+
+LOG_MODULE_REGISTER(main);
+
+#define ADC_NODE DT_NODELABEL(mcp3561)
+
+#define LED_NODE DT_ALIAS(led)
+const struct gpio_dt_spec led = GPIO_DT_SPEC_GET(LED_NODE, gpios);
+
+/* Data of ADC io-channels specified in devicetree. */
+
+static const struct adc_dt_spec adc_chan0 =
+    ADC_DT_SPEC_GET_BY_IDX(DT_PATH(zephyr_user), 0);
+
+int init_led();
```

Comment:
Add comments separating these to be consistent with other main function containing files, [like this](https://github.com/RIT-Launch-Initiative/FSW/blob/6380e0c2756d3c1af55eb490ce9de65332bd9045/app_sensor_module/src/main.c#L19), [or this](https://github.com/RIT-Launch-Initiative/FSW/blob/6380e0c2756d3c1af55eb490ce9de65332bd9045/app_radio_module/src/transmitter.c#L23).